playbooks/kafka_broker.yml

```diff
@@ -31,6 +31,17 @@
     - import_role:
         name: kafka_broker
 
+- name: Kafka Broker Serial Provisioning
```

Comment:
Why do we need to bypass dynamic hosting in case of kraft

Reply:
What we are doing in dynamic provisioning is getting which node is the controller, and updating it separately. 
Here in kraft, none of the brokers will be a controller and all brokers are the same, hence we are rolling the changes in the brokers with serial=1.

Reply:
We have a task pending where we will change our approach to get the controller (not using zookeeper)
After that, we will move back to the approach to run non_controllers first and controllers later (even in kraft). So the code would be more clean.